.gitignore

```diff
@@ -85,3 +85,5 @@ target
 
 notebooks/*.html
 notebooks/*.png
+
+test/
```

Comment:
Is it strictly necessary ? I wonder if it could interfere with unit tests in the future.

Reply:
Indeed it is not necessary you can remove this line

Reply:
Thanks!